### PR TITLE
Exercise edit

### PR DESCRIPTION
**Pull request by andyschulzz**

### CODE REVIEW

File: src/components/Exercise/ExerciseAdd.js [hunk 1]
```diff
@@ -1,32 +1,40 @@
...
+              mla
+              disabled={
+                !formState.dirty || (formState.dirty && !formState.isValid)
+              }
+              type="submit"
+            >
+              Save
+            </Button>
+          )) || (
+            <Button primary mla>
```
Comment (gadmel) [inline src/components/Exercise/ExerciseAdd.js]: What's primary mla?

File: src/components/Exercise/ExerciseAdd.js [hunk 2]
```diff
@@ -60,31 +66,17 @@ export default function ExerciseAdd({ handleExerciseAdd }) {
...
           required
+          disabled={disabled}
         />
       </Styled.Wrapper>
     </section>
   )
   function handleAdd(data) {
-    console.log(data)
-    // handleExerciseAdd(name, type, instructions)
+    Object.keys(data).map(key => (data[key] = data[key].trim()))
```
Comment (gadmel) [inline src/components/Exercise/ExerciseAdd.js]: Cool! 👍

File: src/components/Exercise/ExerciseEdit.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+          name="instructions"
+          id="instructions"
+          required
+          disabled={disabled}
+        />
+      </Styled.Wrapper>
+    </section>
+  )
+
+  function handleChange(data) {
```
Comment (gadmel) [inline src/components/Exercise/ExerciseEdit.js]: 👍

File: src/components/Exercise/ExerciseEdit.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+ExerciseAdd.propTypes = {
+  handleExerciseChange: PropTypes.func.isRequired,
+}
+
+export default function ExerciseAdd({
+  handleExerciseChange,
+  selectedExercise = {},
+}) {
+  const [disabled, setDisabled] = useState(false)
+  const { register, handleSubmit, formState } = useForm({
```
Comment (gadmel) [inline src/components/Exercise/ExerciseEdit.js]: Das müsst ich auch mal so machen!

File: src/components/Exercise/ExerciseAdd.js (hunk 1 above)
Comment (andyschulzz) [reply src/components/Exercise/ExerciseAdd.js]: these are styled component button props for color and a utility prop for margin left auto

Comment (gadmel) [reply src/components/Exercise/ExerciseAdd.js]: styled component button props... that's cool!

**Pull request merged**